build/ci/wait_for_k8s_ready.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -xe
+set +o pipefail
+
+is_kubernetes_cluster_ready (){
+  pods=`kubectl get pods -A | awk '{print$3}' | grep -iv ready`
+  all_the_containers_are_runninig=true
```

Comment:
it's safer to start with `false` and later set to true only when the check passed

Reply:
done